### PR TITLE
refactor: PE-411 repository structure should be reorganized to make initial impression simple

### PR DESCRIPTION
**Pull request by jdmedlin1**
## 📝 Description
The current layout of the repository is not friendly for newcomers; having all templates, examples, configs, etc. thrown at the user at once. 

I propose we categorize the projects into their respective directories for the sake of simplifying the variance of types of info at once. Templates and examples should be put in their own repositories. 

With this change, we will need to adjust npm scripts to point to the same projects. Additionally, we will need to give each new directory its own README as defined in https://brightsign.atlassian.net/browse/PE-410

## 📋 List of Changes

- Please provide a concise list of the main changes.

## 🧪 Steps to Test

1. Step 1

## ✔️ Dev Complete Checklist

- [ ] PR template filled out
- [ ] Change is tested by submitter
- [ ] PR follows all linting and coding standards
- [ ] Github Issue exists (if applicable)
- [ ] Team member has been assigned
- [ ] At least one commit message is in [Conventional Commit](https://www.conventionalcommits.org/) format

### CODE REVIEW

**Review by lherlein — APPROVED** [review]

File: examples/starter-html-example/README.md
Comment (bhastings1019) [inline examples/starter-html-example/README.md:1]: The order of the folder naming convention makes a difference. Should the ordering of the `/*examples*` directories be reconciled? e.g., /example-node-starter-server, /example-html-starter

or something along those lines

**Comment by bhastings1019** [comment]
Is this Draft PR ready for review?

File: README.md [hunk 1]
```diff
@@ -16,75 +16,43 @@ Welcome to the BrightSign Dev Cookbook! Here you will find example code and proj
...
-## Examples
-
-### Overview
-
-The examples in this repository are designed to help you get started with developing on a BrightSign player. These examples serve as stepping stones, allowing you to build and expand your understanding of the platform.
-
-### Learning the basics with examples
+## Learning the basics with examples
 
 If you've never worked with a BrightSign before, we recommend starting with a [barebones example](simple_html_example).
```
Comment (bhastings1019) [inline README.md:28]: update to link to the example we're suggesting

**Review by bhastings1019 — APPROVED** [review]
LGTM

**Comment by bhastings1019** [comment]
The root README may benefit from refining the written form to present HTML (JavaScript) or Node Application, with direct links to each near the top. 

We'll look back over some of these READMEs

**Comment by jdmedlin1** [comment]
Just need a reapproval on this since GitHub is blocking it.

**Review by lherlein — APPROVED** [review]

**Pull request merged**